sympy/utilities/lambdify.py

```diff
@@ -797,10 +803,19 @@ def _lambdifygenerated(x):
                            'allow_unknown_functions': True,
                            'user_functions': user_functions})
 
+    if isinstance(args, set):
+        SymPyDeprecationWarning(
+                    feature="The list of arguments is a `set`. This leads to unpredictable results",
+                    useinstead=": Convert set into list or tuple",
```

Comment:
Why is there a colon here?

Reply:
I guess it's because the template is "use {useinstead} instead". I think we should modify the template so that the `useinstead` is a single standalone sentence. But this would also mean modifying existing deprecation warnings, so it can be done in a separate PR.

Reply:
yes, the colon was to make it more readable once it appear in the notebook or console